### PR TITLE
XS✔ ◾ Rules to better reporting services - Delete duplicate rule

### PR DESCRIPTION
**Pull request by jeoffreyfischer**
**Tip: Use [SSW Rule Writer GPT](https://chat.openai.com/g/g-cOvrRzEnU-ssw-rules-writer) for help with writing rules 🤖**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️ A conversation with Adam - related email, subject "SSW Rules - Delete duplicate rule"

> 2. What was changed?

✏️ The duplicate rule was deleted and the existing rule was archived

> 3. Did you do pair or mob programming (list names)?

✏️No

### CODE REVIEW

File: rules/five-user-experiences-of-reporting-services/rule.md [hunk 1]
```diff
@@ -1,39 +1,37 @@
...
-redirects: []
+created: 2013-07-02T07:51:10.0000000Z
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Lei Xu
+    url: https://ssw.com.au/people/lei-xu
+related: []
+redirects:
+  - five-user-experiences-of-reporting-services
```
Comment (JackDevAU) [inline rules/five-user-experiences-of-reporting-services/rule.md:15]:
```suggestion
  - do-you-know-the-5-user-experiences-of-reporting-services-vanilla-website-email-windows-and-sharepoint
```

File: rules/five-user-experiences-of-reporting-services/rule.md [hunk 2]
```diff
@@ -1,39 +1,37 @@
...
-    url: https://ssw.com.au/people/jeoffrey-fischer
-related:
-- customization-do-you-know-which-version-of-sql-reporting-services-and-visual-studio-you-are-using
-redirects: []
+created: 2013-07-02T07:51:10.0000000Z
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Lei Xu
+    url: https://ssw.com.au/people/lei-xu
```
Comment (JackDevAU) [inline rules/five-user-experiences-of-reporting-services/rule.md:12]: Add yourself as well :)

**Review by JackDevAU — APPROVED** [review]
looks good 👌

Just an FYI - you don't need to add the rules uri into the redirect. Only add different uris into the redirect. In this case the old rule uri.

File: rules/five-user-experiences-of-reporting-services/rule.md (hunk 2 above)
Comment (jeoffreyfischer) [reply rules/five-user-experiences-of-reporting-services/rule.md:12]: I am just migrating the rules, I don't think I should be an author

**Pull request merged**